scripts/v2/post.js

```diff
@@ -799,7 +799,7 @@ function decorateAnimations() {
 
       const $video=createTag('video', attribs);
       if (href.startsWith('https://hlx.blob.core.windows.net/external/')) {
-        href='/hlx_'+href.split('/')[4].replace('#image','');
+        href='/hlx_'+href.split('/')[4].replace('#image','').replace('#video', '');
```

Comment:
would probably better to make this more robust with `new URL(href).pathname` to get rid of all fragments